crypto/secp256k1/secp256k1.go

```diff
@@ -140,6 +144,10 @@ func (pubKey PubKey) String() string {
 	return fmt.Sprintf("PubKeySecp256k1{%X}", []byte(pubKey))
 }
 
+func (pubKey PubKey) Type() string {
+	return "secp256k1"
```

Comment:
Should we have constants for these? Also, the merged JSON changes now register these keys via `RegisterType()`, so we'll still need type identifiers - not sure how that affects these changes.